ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/PipelineChainDownloader.java

```diff
@@ -139,7 +137,7 @@ public synchronized void cancel() {
   private void logDownloadFailure(final String message, final Throwable error) {
     final Throwable rootCause = ExceptionUtils.rootCause(error);
     if (rootCause instanceof CancellationException || rootCause instanceof InterruptedException) {
-      LOG.trace(message, error);
+      LOG.debug(message, error);
     } else if (rootCause instanceof EthTaskException) {
```

Comment:
Seems like we can join the two branches now

Reply:
right, done